### ISSUE TITLE
toolbar not available on load

### ISSUE THREAD

**Issue by csantero** [DockYard/qunit-notifications#4]
I'm using this library in my ember-cli app through ember-cli-qunit, and I am getting

```
Uncaught TypeError: Cannot read property 'appendChild' of undefined
```

on the line where it is trying to append the notification checkbox to the toolbar. Apparently the `qunit-testrunner-toolbar` div has not yet been attached to the DOM. I started getting this error last week when I upgraded my ember-cli to master and started using the ember-cli-qunit addon.

**Comment by sukima** [comment]
This issue needs to be reopened because commit 983434c235f65045a39c461849e26960913144e undoes the fix from #5 and the problem is back.

**Comment by sukima** [comment]
* User-Agent: `Mozilla/5.0 (Macintosh; Intel Mac OS X 10.10; rv:36.0) Gecko/20100101 Firefox/36.0`
* qunit-notification version from `bower_components/qunit-notifications/bower.json`: **0.0.4**

![screen shot 2015-01-13 at 10 46 28 am](https://cloud.githubusercontent.com/assets/70075/5723391/b456261e-9b11-11e4-86ac-ac4d786a04bb.png)

**Comment by bcardarella** [comment]
@amercier would you mind taking a look? I'm slammed today. I can add you to the repo as well if you'd like

**Comment by amercier** [comment]
@bcardarella I won't be able to do this today either, but I'll try to have a look ASAP (maybe tonight after work). Please add me to the repo :) Otherwise I can keep pushing code through PRs.

**Comment by sukima** [comment]
I just `rm -rf bower_components/ node_modules/` then reinstalled. It's working now. I don't understand, I checked the `qunit-notification` code and there was no change. Maybe qunit itself got updated?

Anyway this is still closed. My apologies for the red herring.

**Comment by amercier** [comment]
There was a recent update to QUnit (v1.16.0). PR #7 addressed an incompatibility bug with v1.16, but apparently broke compatibility with v1.15.

However, qunit-notifications should be compatible with both v1.16 and v1.15, if possible. I'll have a look when I have time.

**Comment by amercier** [comment]
Moved to #17
